DIY Reminders with Azure and Twilio/Solution/call.js

```diff
@@ -0,0 +1,45 @@
+//set up client with twilio credentials for this account
```

Comment:
The code looks really well commented! For the commenting style, can you add a space after the // and capitalize the first word of the comment?

Reply:
mkay